extensions/nginx/index.js

```diff
@@ -115,7 +125,10 @@ class NginxExtension extends cli.Extension {
                 return Promise.fromNode(cb => dns.lookup(parsedUrl.hostname, {family: 4}, cb)).catch((error) => {
                     if (error.code !== 'ENOTFOUND') {
                         // Some other error
-                        return Promise.reject(error);
+                        return Promise.reject(new cli.errors.CliError({
+                            message: `Error trying to looking DNS for '${parsedUrl.hostname}'`,
```

Comment:
slight wording fix - should be `Error trying to lookup DNS for '${parsedUrl.hostname}'`

Reply:
Oopsi...